src/main/java/uk/gov/hmcts/darts/authentication/config/AuthenticationConfiguration.java

```diff
@@ -10,4 +11,37 @@
 @EnableConfigurationProperties
 @Getter
 public class AuthenticationConfiguration {
```

Comment:
I find it less confusing if the custom properties we add for our integration outside of what properties are offered by spring, if we don't use their spring prefix. We can use a `@ConfigurationProperties` and have a Properties POJO instead, that might make unit testing easier rather than using `@Value` annotations?

https://docs.spring.io/spring-boot/docs/3.0.5/reference/htmlsingle/#features.external-config.typesafe-configuration-properties.java-bean-binding

Reply:
I intentionally didn't use it. The reason is we are maintaining a single application.yaml which has the security variable variables defined in this format:
spring:
   security:
      oauth2: ... and so on
We would have ended up writing lots of inner classes if we have to use the ConfigurationProperties.

This config will be used by a BaseConfig POJO eventually once we bring in the properties for other ADs. So at runtime based on the request, relevant AD properties will be fethced.

Reply:
OK based on your answer, I think this means that we can remove `@EnableConfigurationProperties` as you aren't using the annotated beans